Sources/SkeletonView.swift

```diff
@@ -183,6 +186,9 @@ extension UIView {
 
     private func recursiveHideSkeleton(reloadDataAfter reload: Bool, transition: SkeletonTransitionStyle, root: UIView? = nil) {
         guard isSkeletonActive else { return }
+        if isHiddenWhenSkeletonIsActive {
```

Comment:
I'm not sure that the skeleton is active, then the if block will never be called, no?

Reply:
Yes it will not, that's why I've called it WhenSkeletonIsActive

Reply:
sorry, I think I didn't explain well. I was meaning the guard condition, `isSkeletonActive` must be true to check the if-block, and I'm not sure if `isSkeletonActive` is true for views that are hidden when the skeleton is active. Do you know what I mean?

Reply:
Wired, actually, isSkeletonActive = true, even when the view turned to hidden in recursiveShowSkeleton method

Do you check if the view is hidden and then start the skeleton process for it?

Reply:
yep, I run your code and I found a problem :/ 

The code only hides the skeletonable views (`isSkeletonable == true`) and the `isHiddenWhenSkeletonIsActive == true`. Why? makes sense, when the skeleton appears, we use the method `subviewsSkeletonables` and this array doesn't contain the view that we need to hide, just the views that we need to convert to the skeleton. 

So, we have two alternatives...maintain this logic, I mean, isSkeletonable must be `true` or...we need to do the same with the views to hide than the views to skeletonize...but I think it's harder because we need to take into account the transitions.

I think we could maintain this logic and specify this behaviour clearly in the README file, WDYT?

Reply:
Yeah, I think it's ok to maintain this logic and explain it clearly. in the README file, in the future, if we needed to add something like this I think it's ok to refactor, but for now, one property is not a problem.